examples/s1_generic.py

```diff
@@ -0,0 +1,309 @@
+
+    return None, None
+
+
+def get_analysis_family_by_family_id(analysis: Analysis, family_id: str) -> Optional[int]:
+    reused_gene_count = None
+    for sub_analysis in analysis.get_sub_analyses():
+        if not sub_analysis.code_reuse:
+            continue
+        for family in sub_analysis.code_reuse['families']:
+            if family['family_id'] == family_id:
+                reused_gene_count = family['reused_gene_count']
+        if reused_gene_count:
+            break
+    return reused_gene_count
+
+
+def find_largest_family(analysis: Analysis) -> dict:
+    largest_family_by_software_type = collections.defaultdict(lambda: {'reused_gene_count': 0})
+    for sub_analysis in analysis.get_sub_analyses():
+        for family in sub_analysis.code_reuse['families']:
+            software_type = family['software_type']
+            if family['reused_gene_count'] > largest_family_by_software_type[software_type]['reused_gene_count']:
+                largest_family_by_software_type[software_type] = family
+    return largest_family_by_software_type
+
+
+def human_readable_size(num: int) -> str:
+    for unit in ['', 'KB', 'MB', 'GB']:
+        if abs(num) < 1024.0:
+            return f'{num:3.1f}{unit}'
+        num /= 1024.0
+    return f'{num:.1f}GB'
+
+
+def get_note(analysis: Analysis) -> str:
+    result = analysis.result()
+
+    metadata = analysis.get_root_analysis().metadata
+    verdict = result['verdict'].lower()
+    sub_verdict = result['sub_verdict'].lower()
+
+    note = (f'Intezer Analyze File Scan\n'
+            f'=========================\n\n')
+
+    if verdict == 'malicious':
+        emoji = '🧨'
+        main_family, gene_count = get_analysis_family(analysis, ['malware', 'malicious_packer'])
+    elif verdict == 'trusted':
+        emoji = '🟢'
+        main_family, gene_count = get_analysis_family(analysis, ['application', 'library', 'interpreter', 'installer'])
+    elif verdict == 'suspicious':
+        emoji = '⚠'
+        main_family, gene_count = get_analysis_family(analysis, ['administration_tool', 'packer'])
+    else:
+        emoji = '❔'
+        main_family = None
+        gene_count = None
+
+    note = f'{note}{emoji} {verdict.capitalize()}'
+
+    if main_family:
+        note = f'{note} - {main_family}'
+        if gene_count:
+            note = f'{note} ({gene_count} shared code genes)'
+    note = f'{note}\n\nSize: {human_readable_size(metadata["size_in_bytes"])}\n'
+    if 'file_type' in metadata:
+        note = f'{note}File type: {metadata["file_type"]}\n'
+    if verdict == 'malicious':
+        iocs = len(analysis.iocs['files']) + len(analysis.iocs['network']) - 1
+
+        if iocs:
+            note = f'{note}IOCs: {iocs} IOCs\n'
+        dynamic_ttps = len(analysis.dynamic_ttps)
+
+        if dynamic_ttps:
+            note = f'{note}TTPs: {dynamic_ttps} techniques\n'
+
+    else:
+        note = f'{note}Verdict: {verdict}\nSub-verdict: {sub_verdict}\n'
```

Comment:
sub verdict should appear after the verdict so it would be 'Suspicious - Administration Tool - Family name'.
it also should appear only in unknown and suspicious. here it shown also for trusted

Reply:
Done